### PR TITLE
UI/Add double attribution chart to current

### PR DESCRIPTION
**Pull request by hellobontempo**
### Adds the separated new/total attribution chart to the current tab

attribution-split

**TO DO**
- [ ] Update CSV to include new counts - will be in a separate PR

### CODE REVIEW

File: ui/app/serializers/clients/monthly.js [hunk 1]
```diff
@@ -29,22 +33,28 @@ export default class MonthlySerializer extends ApplicationSerializer {
...
+    // if object only has outdated key names, update naming
+    if (Object.keys(object).includes('distinct_entities')) {
+      let { clients, distinct_entities, non_entity_tokens } = object;
+      return {
+        clients,
+        entity_clients: distinct_entities,
+        non_entity_clients: non_entity_tokens,
+      };
+    }
+    // TODO CMB: test what to return if neither key exists
```
Comment (hashishaw) [inline ui/app/serializers/clients/monthly.js:57]: I think this TODO is handled by returning the original object

File: ui/app/templates/components/clients/horizontal-bar-chart.hbs [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+  <svg
+    data-test-horizontal-bar-chart
+    class="chart is-horizontal"
+    {{on "mouseleave" this.removeTooltip}}
+    {{did-insert this.renderChart @dataset}}
+    {{did-update this.renderChart @dataset}}
+  >
+  </svg>
+{{else}}
+  <div class="chart-empty-state">
```
Comment (hashishaw) [inline ui/app/templates/components/clients/horizontal-bar-chart.hbs:11]: nice!

File: ui/mirage/handlers/clients.js [hunk 3]
```diff
@@ -663,6 +664,7 @@ const handleMockQuery = (queryStartTimestamp, monthlyData) => {
     do {
       i++;
       let timestamp = formatRFC3339(sub(startDateByMonth, { months: i }));
+      // this is probably not what the empty object looks like but waiting to hear back from backend
```
Comment (hashishaw) [inline ui/mirage/handlers/clients.js:667]: Can we add a "TODO" here so we make sure to come back and clean it up once we do hear back from backend?

File: ui/tests/helpers/clients.js [hunk 4]
```diff
@@ -58,7 +58,7 @@ export function generateConfigResponse(overrides = {}) {
     request_id: 'some-config-id',
     data: {
       default_report_months: 12,
-      enabled: 'default-enable',
```
Comment (hashishaw) [inline ui/tests/helpers/clients.js:61]: FYI, when ENT vault is started this value is `default-enable` from the API until it's changed, and likewise for OSS and `default-disable`. We should make sure we don't lose support for those values (unless the API removed them and I wasn't aware)

**Review by hashishaw — APPROVED** [review]
Nice work! A couple small comments but nothing blocking

File: ui/mirage/handlers/clients.js (hunk 3 above)
Comment (hellobontempo) [reply ui/mirage/handlers/clients.js:667]: Great idea!

File: ui/tests/helpers/clients.js (hunk 4 above)
Comment (hellobontempo) [reply ui/tests/helpers/clients.js:61]: Oh! I has looked and saw it was `enable` but I believe that was when I had turned it off and back on....I'll change back!

**Pull request merged**